packages/db/prisma/schema.prisma

```diff
@@ -17,8 +17,17 @@ model Developers {
 }
 
 model Posts {
-  id        Int      @id @default(autoincrement())
+  id        Int        @id @default(autoincrement())
   title     String
   contents  String
+  createdAt DateTime   @default(now())
+  comments  Comments[]
+}
+
+model Comments {
```

Comment:
Can we rename these to be singular (my mistake in the tickets)? so Post and Comment -- I think that will make more sense when you have something like `Comment[]`. Also makes more sense that a comment has only one post, but right now it is type `Posts` which is confusing.

Reply:
Very fair point